app/models/shipit/stack.rb

```diff
@@ -240,21 +240,23 @@ def status
     end
 
     def lock_reverted_commits!
```

Comment:
@casperisfine thoughts on this change? I refactored the logic to lock in batches rather than all at once so the `lock_author` can be set to the revert commit author.

Tests are gonna fail. Will fix after work.

Reply:
LGTM. I'd use `affected_rows` rather than `affected` though /nitpick.